RELEASES.md

```diff
@@ -83,14 +82,18 @@ Please respond LGTM or REJECT (with reasoning).
 
 $sig
 ```
-* [ ] edit/update the pull-request to link to the VOTE thread, from https://groups.google.com/a/opencontainers.org/forum/#!forum/dev
+
+* [ ] edit/update the pull-request to link to the VOTE thread, from <https://groups.google.com/a/opencontainers.org/forum/#!forum/dev>
 * [ ] a week later, if the vote passes, merge the PR
   * [ ] `git tag -s $version $versionBumpCommit`
   * [ ] `git push --tags`
 * [ ] produce release documents
   * [ ] git checkout the release tag, like `git checkout $version`
   * [ ] `make docs`
   * [ ] rename the output PDF and HTML file to include version, like `mv output/oci-runtime-spec.pdf output/oci-runtime-spec-$version.pdf``
-  * [ ] attach these docs to the release on https://github.com/opencontainers/runtime-spec/releases
+  * [ ] attach these docs to the release on <https://github.com/opencontainers/runtime-spec/releases>
   * [ ] link to the the VOTE thread and include the passing vote count
   * [ ] link to the pull request that merged the release
+* [ ] add release notes to the website <https://github.com/opencontainers/opencontainers.org/tree/main/content/release-notices>
```

Comment:
The IP stuff seems complicated https://github.com/opencontainers/opencontainers.org/blob/main/content/release-notices/v1-0-2-runtime-spec.md , not sure if this should be written by spec maintainers

Reply:
Most of it looks like a copy/paste from previous releases. @amye may know more about what maintainers should handle vs LF.